app/views/registration_wizard/shared/questions/_text_field.html.erb

```diff
@@ -2,6 +2,6 @@
   form.govuk_text_field(
     question.name,
     width: "full",
-    label: { size: "xl", tag: "h1" }
+    label: { text: question.question_text, size: "xl", tag: "h1" }
```

Comment:
This had to be set like this rather than relying on the built in auto-translation because in this questions case the question name (`institution_location`) is used in multiple questions but with different question copy. `question_text` uses the `name_locale_key` method off of Forms::QuestionTypes::Base to allow you to override the key used for the text locale lookup.